sdk/src/types/block/context_input/mod.rs

```diff
@@ -87,7 +87,7 @@ pub(crate) type ContextInputCount =
 #[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Deref, Packable)]
 #[packable(unpack_error = Error, with = |e| e.unwrap_item_err_or_else(|p| Error::InvalidContextInputCount(p.into())))]
 pub struct ContextInputs(
-    #[packable(verify_with = verify_context_inputs_packable)] BoxedSlicePrefix<ContextInput, ContextInputCount>,
+    #[packable(verify_with = verify_context_inputs)] BoxedSlicePrefix<ContextInput, ContextInputCount>,
```

Comment:
Just making sure, since `ContextInputs` doesn't have a packable `unpack_visitor` attribute, it'll still be forwarded a `()` and then calling the verify if the parent type does have a visitor, right?

Reply:
Basically, yes